bin/dockerUtils.js

```diff
@@ -0,0 +1,252 @@
+// Inquirer used to Create Prompts in Terminal.
+import inquirer from "inquirer";
+
+export async function promptUserForServices() {
+  const services = [];
+
+  // Aggregating the answers for app service
+  const appAnswers = await inquirer.prompt([
+    {
+      type: "input",
+      name: "serviceName",
+      message: "Enter the name of the app service (default: app):",
+      default: "app",
+    },
+    {
+      type: "input",
+      name: "image",
+      message: "Enter the Docker image for the app service (default: node:20-alpine):",
+      default: "node:20-alpine",
+    },
+    {
+      type: "input",
+      name: "containerName",
+      message: "Enter the container name for the app service (default: app_container):",
+      default: "app_container",
+    },
+    {
+      type: "input",
+      name: "ports",
+      message: "Enter the ports to expose for the app service (default: 8080:8080):",
+      default: "8080:8080",
+      validate: (input) => {
+        const validFormat = /^\d+:\d+$/;
+        return validFormat.test(input) || "Ports must be in the format hostPort:containerPort.";
+      },
+    },
+    {
+      type: "confirm",
+      name: "addEnvironmentVariables",
+      message: "Do you want to add environment variables for the app service?",
+      default: false,
+    },
+    {
+      type: "input",
+      name: "environmentVariables",
+      message: "Enter environment variables (comma-separated, e.g., VAR1=value1,VAR2=value2):",
+      when: (answers) => answers.addEnvironmentVariables,
+      validate: (input) => {
+        return input.split(",").every((env) => env.includes("=")) || "Each variable must be in VAR=value format.";
+      },
+    },
```

Comment:
`.env` directly from `.env` in template. We'll be making sure it exists soon. So no need to get this way ig. I'm not sure. What do you say?